root/static/scripts/relationship-editor/utility/compareRelationships.js

```diff
@@ -352,6 +365,11 @@ export default function compareRelationships(
     return datePeriodCmp;
   }
 
+  const timeCmp = compareStrings(getTimeIfPresent(a), getTimeIfPresent(b));
```

Comment:
Looks like this function doesn't take text attribute values into account in general, other than series numbers above, and now "time" here (unless I'm missing something).

I can't remember if there's a reason `compareAttributeLists` (via `compareNullableLinkAttributes`) uses `compareLinkAttributeIds` (which only compares the IDs) and not `compareLinkAttributes` (which compares the text values too), but I'd think it should use the latter...otherwise we may have issues with other text attributes.

Reply:
I think it might be intentional that we want some text attributes to be considered duplicates? I'd have to review all our uses of text attributes before confidently agreeing to split them.

Reply:
Makes sense, I was basing my comment on `Data::Link::find`, which seems to consider all text attributes for differentiation currently (but there are probably some blocked by the UI and not the server).